themes/bootstrap3/templates/ajax/status-full.phtml

```diff
@@ -30,26 +30,16 @@
         <?php endif; ?>
       </td>
       <td class="fullAvailability">
-        <?php if ($item['use_unknown_message'] ?? false): ?>
-          <span class="text-muted"><?=$this->transEsc('status_unknown_message')?></span>
-        <?php elseif ((int)$item['availability'] === \VuFind\ILS\Logic\ItemStatus::STATUS_UNAVAILABLE): ?>
-          <span class="text-danger"><?=$this->transEsc($item['status'])?></span>
-        <?php elseif ((int)$item['availability'] === \VuFind\ILS\Logic\ItemStatus::STATUS_UNCERTAIN): ?>
-          <span class="text-warning"><?=$this->transEsc($item['status'])?></span>
-        <?php else: ?>
-          <span class="text-success">
-            <?php
-              if ($item['reserve'] === 'Y') {
-                $status = 'On Reserve';
-              } elseif (is_int($item['availability']) && !empty($item['status'])) {
-                $status = $item['status'];
-              } else {
-                $status = 'Available';
-              }
-            ?>
-            <?=$this->transEsc($status)?>
-          </span>
-        <?php endif; ?>
+        <?php $availabilityStatus = $item['availability']; ?>
+          <?php
+            $statusClassEsc = $this->availabilityStatus()->getClass($availabilityStatus);
+            if ($availabilityStatus->isAvailable() && $item['reserve'] === 'Y') {
```

Comment:
I imagine we may in future want to make the reserve handling more flexible too -- I can imagine scenarios where we would want to show both "on reserve" status and the present availability of the item, which current mechanisms do not allow. No action needed now, but something to think about....